tests/test_license_expression.py

```diff
@@ -552,6 +552,14 @@ def test_parse_simplify_and_contain_and_equal(self):
         expr3 = licensing.parse('mit and LGPL2.1')
         self.assertTrue(expr3 in expr2)
 
+    def test_parse_simplify_no_sort(self):
+        licensing = Licensing()
+
+        expr = licensing.parse('gpl-2.0 OR apache-2.0 OR gpl-2.0')
+
+        self.assertEqual(expr.simplify(), expr.simplify(sort=False))
```

Comment:
with `gpl-2.0 OR apache-2.0 OR gpl-2.0` I would expect the simplification to be `gpl-2.0 OR apache-2.0` ... would you?

Also a small nitpicking: may be we do not need so many empty lines?

Reply:
This is checking to make sure the expressions themselves are equal, even if the "sort" of the expression is different. For instance `mit OR gpl-2.0 === gpl-2.0 OR mit` are technically the same data-wise and interpretation-wise. 

The only difference between the two expressions is the ordering, which the following test on line 560 handles.

Reply:
@pombredanne and we probably do not need so many lines, I will fix this ;)